cvat/apps/engine/models.py

```diff
@@ -583,23 +583,27 @@ def __str__(self):
 
 class CloudStorage(models.Model):
     # restrictions:
-    # AWS bucket name, Azure container name - 63
+    # AWS bucket name, Azure container name - 63, Google bucket name - 63 without dots and 222 with dots
+    # https://cloud.google.com/storage/docs/naming-buckets#requirements
     # AWS access key id - 20
     # AWS secret access key - 40
     # AWS temporary session tocken - None
     # The size of the security token that AWS STS API operations return is not fixed.
     # We strongly recommend that you make no assumptions about the maximum size.
     # The typical token size is less than 4096 bytes, but that can vary.
+    # specific attributes:
+    # location - max 23
+    # project ID: 6 - 30 (https://cloud.google.com/resource-manager/docs/creating-managing-projects#before_you_begin)
     provider_type = models.CharField(max_length=20, choices=CloudProviderChoice.choices())
-    resource = models.CharField(max_length=63)
+    resource = models.CharField(max_length=222)
```

Comment:
@nmanovic, I've increased restrictions for bucket names length because google cloud storage has 2 restriction [variants](https://cloud.google.com/storage/docs/naming-buckets#requirements) - 63 like other providers and 222 with dots in bucket name.  Are we going to support the second case?

Reply:
@Marishka17 , you can even have 255 (just in case). I really don't think that we have to be very strict here.